demo/common/assets.js

```diff
@@ -781,6 +781,14 @@ shakaAssets.testAssets = [
       .addFeature(shakaAssets.Feature.LIVE)
       .addFeature(shakaAssets.Feature.SUBTITLES)
       .addFeature(shakaAssets.Feature.MP4),
+  new ShakaDemoAssetInfo(
+      /* name= */ 'Low Latency DASH Live',
```

Comment:
Since we already have a 'Low Latency DASH Live' manifest, would you please change it to another name?

Reply:
Done!